modules/openapi-generator/src/main/resources/rust/reqwest/api.mustache

```diff
@@ -90,6 +86,19 @@ impl {{{classname}}} for {{{classname}}}Client {
         {{/authMethods}}
         {{/hasAuthMethods}}
 
+        {{#hasFormParams}}
+        let mut form_params = HashMap::new();
+        {{#formParams}}
+        {{#isFile}}
+        form_params.insert("{{{baseName}}}".to_string(), unimplemented!());
```

Comment:
I've given this a quick skim. As a thought, when do we want to fail if a user does attempt to create a generator with a file form parameter? This `unimplemented` would mean that we would fail at run-time when that endpoint was called.

Might it be worth using `compile_error!` (https://doc.rust-lang.org/std/macro.compile_error.html) so that the user discovers this at compile time instead?

Reply:
`compile_error!("File form parameters not supported by the rust generator");`, perhaps?

Reply:
I ported the Hyper implementation (cf #525). We can switch to `compile_error!` macro, but we probably should do the same in `hyper` library templates to be consistent.

Reply:
I tested `compile_error!` macro and I am questioning…
Since petstore example contains a file form param, with this macro, generated sample code does not compile anymore: the project cannot be used anymore to check syntax error regressions, for example.

So, I suggest keeping `unimplemented!`.